rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -226,9 +226,10 @@ impl PrometheusVm {
                     continue;
                 }
 
-                let file_name = format!("{}.json", file.file_name().as_os_str().to_str().unwrap());
+                let file_name = file.file_name();
+                let file_name = file_name.as_os_str().to_str().unwrap();
```

Comment:
Why not just inline this to:
```suggestion
                let file_name = file.file_name().as_os_str().to_str().unwrap();
```

Reply:
```bash
    Checking ic-system-test-driver v0.9.0 (/ic/rs/tests/driver)
error[E0716]: temporary value dropped while borrowed
   --> rs/tests/driver/src/driver/prometheus_vm.rs:230:33
    |
230 |                 let file_name = file.file_name().as_os_str().to_str().unwrap();
    |                                 ^^^^^^^^^^^^^^^^                              - temporary value is freed at the end of this statement
    |                                 |
    |                                 creates a temporary value which is freed while still in use
231 |
232 |                 let destination_path = dashboard_dir.join(file_name);

```

Reply:
Ah ok. Nevermind then.